_pytest/cacheprovider.py

```diff
@@ -8,13 +8,17 @@
 import py
 import pytest
 import json
-from os.path import sep as _sep, altsep as _altsep
+from os.path import sep as _sep, altsep as _altsep, \
+                    isabs as _isabs, expanduser as _expanduser
 
 
 class Cache(object):
     def __init__(self, config):
         self.config = config
-        self._cachedir = config.rootdir.join(".cache")
+        cache_dirname = config.getini("cache_dirname")
```

Comment:
I would rather name this `cache_dir` instead so we can support absolute paths in the future.